templates/releases.hbs

```diff
@@ -31,6 +54,59 @@
     <div class="release">
       <a href="/releases"><strong>Recent Releases</strong></a> <a href="/releases/feed" title="Atom feed"><i class="fa fa-rss-square"></i></a>
     </div>
+    {{else}}
```

Comment:
Why is the JS that registers the key handlers in the `else` branch of the template? Wouldn't that make it not appear on the front page?

Reply:
That's the whole point! I want to be able to navigate through my search results, not on the front page.

Reply:
But doesn't this also register the "S" shortcut?

Reply:
No, the "S" shortcut is "registered" through `handleShortcut` whereas the up and down arrows are "registered" through `handleKey`.

Reply:
Aha, i see it now. I was confused because this also registered "S", but that's probably because the handler overwrites `document.onkeypress`/`document.onkeydown`. This looks fine, then.